src/qt/getaddresstoreceive.cpp

```diff
@@ -73,6 +75,17 @@ void getAddressToReceive::getNewAddress()
     showQR();
 }
 
+void getAddressToReceive::getColdStakingAddress()
+{
+    if (!IsCommunityFundEnabled(pindexBestHeader,Params().GetConsensus()))
```

Comment:
Based on the response in the QMessageBox it seems that this conditional should have used IsColdStakingEnabled.

Reply:
https://github.com/navcoin/navcoin-core/pull/733